README.md

```diff
@@ -128,53 +128,57 @@ functions:
   prefix: 'cx_' # Prefix added to all functions.
   prefix-replacement: # Replaces a functions's prefix.
     'clang_': ''
-    '_': 'C'</pre></td>
+    '_': 'C'</code></pre></td>
   </tr>
   <tr>
     <td>array-workaround</td>
     <td>no</td>
     <td>Should generate workaround for fixed arrays in Structs. See <a href="#array-workaround">Array Workaround</a><br>
       <b>Default: false</b>
     </td>
-    <td><pre lang='yaml'>array-workaround: true</pre></td>
+    <td><pre lang="yaml"><code>array-workaround: true</code></pre></td>
   </tr>
   <tr>
     <td>comments</td>
     <td>no</td>
     <td>Extract documentation comments for declarations.<br>
-      <i>Options: brief/full/none</i><br>
-      <b>Default: brief</b><br>
-      By default clang only parses documentation comments. To enable
-      ordinary comments (starting with // or /*) add<br>
-      <code>-fparse-all-comments</code> to <code>compiler-opts</code>.
+    You can specify the style and length of the comment from one of these options<br>
```

Comment:
The style and length of the comments can be specified with the following options. (remove you)